### PR TITLE
Remove media and audio pickers

### PR DESCRIPTION
**Pull request by IanVS**
And target iphoneos 12, to avoid some warnings. 

We only use the file picker to allow users to load in nebula certs/keys, we don't need access to pictures.

### CODE REVIEW

**Comment by IanVS** [comment]
https://github.com/miguelpruivo/flutter_file_picker/issues/1236

File: ios/Runner/Info.plist [hunk 1]
```diff
@@ -43,8 +43,6 @@
 	<true/>
 	<key>NSCameraUsageDescription</key>
 	<string>Camera permission is required for qr code scanning.</string>
-	<key>NSPhotoLibraryUsageDescription</key>
```
Comment (jasikpark) [inline ios/Runner/Info.plist:46]: is it possible to load a cert from a picture you took earlier of the qr code?

**Review by nbrownus — APPROVED** [review]

File: ios/Runner/Info.plist (hunk 1 above)
Comment (nbrownus) [reply ios/Runner/Info.plist:46]: When this was included we didn't have an option to ignore the photo library

Comment (jasikpark) [reply ios/Runner/Info.plist:46]: so i'm guessing the answer is no then?

Comment (nbrownus) [reply ios/Runner/Info.plist:46]: I'm sure it could be technically possible but its not a flow I am particularly interested in preserving

**Pull request merged**